sympy/polys/ring_series.py

```diff
@@ -572,6 +574,108 @@ def _coefficient_t(p, t):
             p1[monomial_div(expv, expv1)] = p[expv]
     return p1
 
+def rs_series_reversion_newton(p,x,n):
+
+    r"""
+    Fast series reversion (aka Lagrange Inversion) using Newton-like updates.
+
+    Given a ring polynomial $p$ in variable $x$, this method computes a ring polynomial
+    $q$ in $x$ such that $q(p(x)) = x + O(x^n)$.
+
+    The series should be of form $p(x) = a_1 x + a_2 x^2 + ...$ where $a_1$ is invertible.
+
+    Parameters
+    ==========
+
+    p : :class:`~.PolyElement` to be inverted.
+    x : :class:`~.PolyElement` with respect to which ``p`` is inverted.
+    n : number of expansion terms to be computed.
```

Comment:
See the docstring guide for formatting this:
https://docs.sympy.org/latest/contributing/docstring.html#parameters-section

It should be like
```
param: param_type
    A description of what param is.
```